example/pubspec.lock

```diff
@@ -177,7 +177,7 @@ packages:
     source: hosted
     version: "2.1.4"
   wolt_modal_sheet:
-    dependency: "direct dev"
+    dependency: "direct main"
```

Comment:
Should avoid updating `.lock` files in the unrelated for compatibility with the stable channel. 

Only update these files when upgrading from an older stable to a newer stable.

Reply:
Can you clarify this a bit? I would be interested in knowing more about this since I didn't have strong opinion.

Reply:
To be make sure the package is always compatible with the Flutter stable We should make sure .lock files are generated or updated on using the stable channel.  

I can see there is another new file `.lock` file generated.

This PR is for the Top Bar configuration.  So any pubspec or `.lock` files changes should be excluded.

Reply:
But if we don't merge these, they appear in every branch. Do you mean that they need to be aligned in a separate PR?

Reply:
Yes but there is no hard rule.
But I'd suggest to keep them separate since they that have nothing to do with the PR subject and intention. It also makes it much easier to track changes, for example you won't see top bar commit in a pubspec git history, 

This is for now tho.